### PR TITLE
Jest branch

### PR DESCRIPTION
**Pull request by rida-elbahtouri**
### In this branch I Created test for this simple functions:
- capitalize(string)
- reverseString(string)
- calculator(add, subtract, divide, and multiply)
- Caesar Cipher
- Array Analysis

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @rida-elbahtouri, congratulations on a great job :100: All test cases work as expected :+1: Code runs well :slightly_smiling_face: However, let us add 2 more test cases to our `caeser.test.js`
Kindly find the comments below:
- Let us add more test cases to our caeser test file as in the screenshot below:\
fr
- Kindly remove the `lock` files to avoid problems with future versions

Once this is fix, kindly submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team 👋

Nice work on this.👌
However, let's make the following changes:

- [ ] Please note that in the ceasar cipher test you are expected to include a test to check wrapping from z to a. Eg test string: "yellow" then ensure that "y" can be wrapped around to the beginning of the alphabets again. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/caesar_requirement.png).

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/calculator.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const Calculator = (num1, num2) => {
+  const add = () => num1 + num2;
+  const subtract = () => num1 - num2;
+  const divide = () => num1 / num2;
+  const multiply = () => num1 * num2;
+  return {
+    add, divide, subtract, multiply,
+  };
+};
```
Comment (mikenath223) [inline src/calculator.js]: - [ ] Also this is expected to be an object not a constructor function. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/calculator_requirement.png). It should be in this manner:

```
const calculator = {
add: () => {},
...
}
```

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi again Rida 👋

Good job making some of the recommended fixes in the last review.

However, there is just one more left and you would be all set.

- [ ] You were asked to make [this change](https://github.com/rida-elbahtouri/Testing_Jest/pull/1#discussion_r488204682). Please make the needed change.

Kindly fix this issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**